observability/otel/otel-collector/otel-collector-exporters.adoc

```diff
@@ -153,41 +153,42 @@ include::snippets/technology-preview.adoc[]
 [source,yaml]
 ----
 # ...
-  ports:
-  - name: promexporter # <1>
-    port: 8889
-    protocol: TCP
   config: |
     exporters:
       prometheus:
-        endpoint: 0.0.0.0:8889 # <2>
-        tls: # <3>
+        endpoint: 0.0.0.0:8889 # <1>
+        tls: # <2>
           ca_file: ca.pem
           cert_file: cert.pem
           key_file: key.pem
-        namespace: prefix # <4>
-        const_labels: # <5>
+        namespace: prefix # <3>
+        const_labels: # <4>
           label1: value1
-        enable_open_metrics: true # <6>
-        resource_to_telemetry_conversion: # <7>
+        enable_open_metrics: true # <5>
+        resource_to_telemetry_conversion: # <6>
           enabled: true
-        metric_expiration: 180m # <8>
-        add_metric_suffixes: false # <9>
+        metric_expiration: 180m # <7>
+        add_metric_suffixes: false # <8>
     service:
       pipelines:
         metrics:
           exporters: [prometheus]
 # ...
 ----
-<1> Exposes the Prometheus port from the Collector pod and service. You can enable scraping of metrics by Prometheus by using the port name in `ServiceMonitor` or `PodMonitor` custom resource.
-<2> The network endpoint where the metrics are exposed.
-<3> The server-side TLS configuration. Defines paths to TLS certificates.
-<4> If set, exports metrics under the provided value. No default.
-<5> Key-value pair labels that are applied for every exported metric. No default.
-<6> If `true`, metrics are exported using the OpenMetrics format. Exemplars are only exported in the OpenMetrics format and only for histogram and monotonic sum metrics such as `counter`. Disabled by default.
-<7> If `enabled` is `true`, all the resource attributes are converted to metric labels by default. Disabled by default.
-<8> Defines how long metrics are exposed without updates. The default is `5m`.
-<9> Adds the metrics types and units suffixes. Must be disabled if the monitor tab in Jaeger console is enabled. The default is `true`.
+<1> Required: The network endpoint where the metrics are exposed. The {OTELOperator} automatically exposes the port specified in the `endpoint` field to the `<instance_name>-collector` service.
```

Comment:
"Required" here can indicate that all the other parameters are optional. Is this the case?

Reply:
Actually it can be removed